docs/content/topics/project-developers/projects.md

```diff
@@ -9,296 +9,267 @@ aliases:
+(`secrets.yaml`) and a `NOTES.txt` file with next steps.  Additionally, a
+`.gitignore` file is created to ensure that the secrets file and script
+dependencies are not tracked in version control.
 
-### The Brigade Project Chart
+### The `brig project create` Command
 
-The Brigade Project chart is located in the [brigadecore/charts][charts] source tree at
-`charts/brigade-project`. You can also install it out of the Brigade chart repository.
+With a project definition file in hand, you're now ready to create the project
+with brig:
 
 ```console
-$ helm repo add brigade https://brigadecore.github.io/charts
-$ helm search brigade/brigade-project
-NAME                   	CHART VERSION	APP VERSION	DESCRIPTION
-brigade/brigade-project	1.0.0       	v1.0.0    	Create a Brigade project
+$ brig project create --file project.yaml
 ```
 
-We recommend using the following pattern to create your project:
+This command will submit the project definition to Brigade's API server. After
+validating that the definition adheres to the project schema, Brigade will
+persist the project on the substrate (in the form of a unique namespace) and in
+the backing database.
 
-#### 1. Create a place to store project configs
+### Upgrade a project
 
-Store your project configuration in a safe place (probably locally or in a storage
-system like Keybase).
+You can upgrade a project at any time with the following command:
 
 ```console
-$ mkdir -p brigade-projects/myproject
-$ cd brigade-projects/myproject
+$ brig project update --file project.yaml
 ```
 
-> You can store project configs in Git repos, but we don't recommend GitHub for
-> this. Keybase has [free encrypted private Git repos](https://keybase.io/blog/encrypted-git-for-everyone),
-> which are great for this sort of thing.
+### Delete a project
 
-#### 2. Create a `values.yaml` file for your project
+To delete a project, run:
 
+```console
+$ brig project delete --id myproject
 ```
-$ helm inspect values brigade/brigade-project > values.yaml
-```
-
-#### 3. Edit the values for your project
-
-Read through the generated `values.yaml` file and modify it accordingly.
 
-Our suggestions are as follows, replacing `brigadecore/empty-testbed` with your own project's name:
+### Listing and inspecting projects with `brig`
 
-```yaml
-# Definitely do these:
-project: "brigadecore/empty-testbed"
-secrets: {}
+You can list all projects via:
 
-# Probably do these so you can load a GitHub project which has useful stuff
-# in it, and if you want to use GitHub webhooks.
-repository: "github.com/brigadecore/empty-testbed"
-cloneURL: "https://github.com/brigadecore/empty-testbed.git"
-github:
-   token: "github oauth token"
+```console
+$ brig project list
+```
 
-# If you want GitHub webhooks.
-sharedSecret: "IBrakeForSeaBeasts"
+You can also directly inspect your project with `brig project get`:
 
-# As for the rest, use them if you know you need them.
+```console
+$ brig project get --id myproject
```

Comment:
I think it might also be beneficial to explore the `--output` / `-o` flag here since that will allow users to see the _full_ project definition.